src/static/js/modules/form-validation.js

```diff
@@ -50,14 +50,38 @@ function _clearNotification( elem ) {
   $elem.find( '.error' ).removeClass( 'error' );
 }
 
+function _sendSubscriptionRequest( elem ) {
+  event.preventDefault();
```

Comment:
Nope, it's passed in the callback in the plugin (I know it's kinda weird, open to better suggestions).